dev/reference/tools/data-migration/cluster-operations.md

```diff
@@ -489,23 +489,52 @@ Assuming that the `172.16.10.72` machine needs to be maintained or this machine
 
     6. Restart and verify data migration task.
 
-## Switch between master and slave instances
+## Switch DM-worker connection between upstream MySQL instances
```

Comment:
- The following section is a separate file in docs-cn. Please check https://github.com/pingcap/docs-cn/pull/1871 and update.
- Suggest putting it under "Manage the DM Cluster". Please confirm this with @csuzhangxc.